lib/modules/datasource/custom/readme.md

```diff
@@ -1,18 +1,17 @@
-This `custom` datasource allows requesting version data from generic HTTP endpoints.
+This `custom` datasource allows requesting version data from generic HTTP(S) endpoints.
 
 ## Usage
 
 The `customDatasources` option takes a record of `customDatasource` configs.
-This example shows how to update the `k3s.version` file with a custom datasource and
-a [regexManagers](../../manager/regex/):
+This example shows how to update the `k3s.version` file with a custom datasource and a [regexManagers](../../manager/regex/):
```

Comment:
it should link to a md file, otherwise we get warning on docs build

Reply:
I don't see any `lib/modules/manager/regex` folder in my PR branch. I suspect we autogenerate that folder in one of our build steps?

Reply:
yes, it's generated from `lib/modules/manager/custom/regex/readme.md` here

https://github.com/renovatebot/renovate/blob/f37bfb29c32fe097a90ac5b6f1df95ecca279837/tools/docs/manager.ts#L152